### PR TITLE
[PT-BR] Translate Async Await and Immutability

### PR DESCRIPTION
**Pull request by vinicius98s**
Translation for:
- [Async Await](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/playground-examples/copy/en/JavaScript/Modern%20JavaScript/Async%20Await.ts)
- [Immutability](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/playground-examples/copy/en/JavaScript/Modern%20JavaScript/Immutability.ts)

### CODE REVIEW

File: packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Async Await.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
+//// { order: 1, target: "es5" }
+
+// O JavaScript moderno adicionou um modo de lidar com callbacks
+// de uma maneira elegante adicionando uma API baseada em Promise
+// na qual tem uma sintaxe especial que te permite tratar
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Async Await.ts]:
```suggestion
// a qual tem uma sintaxe especial que te permite tratar
```

File: packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Async Await.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
+//// { order: 1, target: "es5" }
+
+// O JavaScript moderno adicionou um modo de lidar com callbacks
+// de uma maneira elegante adicionando uma API baseada em Promise
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Async Await.ts]:
```suggestion
// de uma maneira elegante adicionando uma API baseada em Promises
```

File: packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Async Await.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+// E deixa se tornar linear como:
+
+// const resposta = await pegarResposta(url)
+// const segundaResposta = await pegarResposta(reposta.url)
+// const dadosDaResposta = segundaResposta.data
+// const terceiraResposta = await pegarResposta(dadosDaResposta.url)
+// ...
+
+// No qual pode fazer o código ficar mais perto do canto esquerdo
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Async Await.ts]:
```suggestion
// O qual pode fazer o código ficar mais perto do canto esquerdo
```

File: packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Immutability.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
+// O JavaScript é uma linguagem com algumas maneiras de
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Immutability.ts]:
```suggestion
// O JavaScript é uma linguagem que possui algumas maneiras de
```

File: packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Immutability.ts [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+// declarar que alguns de seus objetos não mudam.
+// O mais proeminente é const - no qual diz que o valor não mudará.
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Immutability.ts]:
```suggestion
// O mais proeminente é const - que diz que o valor não mudará.
```

File: packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Immutability.ts [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+const olaMundo = "Olá Mundo";
+
+// Você não pode mudar olaMundo agora, o TypeScript
+// te dará um erro sobre isso, porque você teria um erro no
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/JavaScript/Modern JavaScript/Immutability.ts]:
```suggestion
// te dará um erro, porque você teria um erro em
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Some typo corrections

**Comment by vinicius98s** [comment]
@khaosdoctor Applied those suggestions

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**